### PR TITLE
update default value of tidb_analyze_version in new versions

### PR DESCRIPTION
**Pull request by yiwen92**
### First-time contributors' checklist 

- [X] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
update default value of tidb_analyze_version in new versions

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [X] master (the latest development version)
- [X] v5.4 (TiDB 5.4 versions)
- [X] v5.3 (TiDB 5.3 versions)
- [X] v5.2 (TiDB 5.2 versions)
- [X] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

this PR is translated from https://github.com/pingcap/docs/pull/7569#issue-1137161527

### CODE REVIEW

**Comment by TomShawn** [comment]
@yiwen92 Could you please sign the CLA as reminded in https://github.com/pingcap/docs/pull/7569#issuecomment-1038971191? Thanks~

**Comment by yiwen92** [comment]
> @yiwen92 Could you please sign the CLA as reminded in [#7569 (comment)](https://github.com/pingcap/docs/pull/7569#issuecomment-1038971191)? Thanks~

Done

File: statistics.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ aliases: ['/docs/dev/statistics/','/docs/dev/reference/performance/statistics/']
 
 # Introduction to Statistics
 
-TiDB uses statistics to decide [which index to choose](/choose-index.md). The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`. In versions before v5.1.0, the default value of this variable is `1`. In v5.1, v5.2, and v5.3, the default value of this variable is `2`, which serves as an experimental feature.
+TiDB uses statistics to decide [which index to choose](/choose-index.md). The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`. In versions lower than v5.1.0, the default value of this variable is `1`, analyze introduce value `2` in v5.1.0. For versions in v5.1.0-v5.1.3 and v5.2.0-v5.2.3, the default value of this variable is `2`, which serves as an experimental feature(In later versions of v5.1.4 and v5.2.4, the default value of this variable is changed to `1`). For all versions later than v5.3.0, the default value of this variable is `2`.
```
Comment (xuyifangreeneyes) [inline statistics.md:9]: Maybe `Since v5.3.0, the default value ....` is better?

Comment (hfxsd) [inline statistics.md:9]:
```suggestion
TiDB uses statistics to decide [which index to choose](/choose-index.md). The `tidb_analyze_version` variable controls the statistics collected by TiDB. Currently, two versions of statistics are supported: `tidb_analyze_version = 1` and `tidb_analyze_version = 2`. The following table describes the default value for each version.

| Version | Default value |
| :--- | :--- |
| < v5.1.0 |  1 |
| v5.1.0 - v5.1.3 |  2 |
| v5.2.0 - v5.2.3 |  2 |
| >= v5.1.4  |  1 |
| >= v5.2.4  |  1 |
| >= v5.3.0  |  2 |
```

**Comment by TomShawn** [comment]
本 PR 翻译自 https://github.com/pingcap/docs-cn/pull/8364。8364 已关闭，本 PR 也需关闭。

**Pull request closed without merging**